Testing/SystemTests/tests/analysis/PredictPeaksTest.py

```diff
@@ -89,9 +89,9 @@ def runTest(self):
                              MinDSpacing=0.5, MaxDSpacing=10,
                              CalculateStructureFactors=True)
 
-        self.assertEquals(peaks.getNumberPeaks(), 540)
+        self.assertEquals(peaks.getNumberPeaks(), 546)
```

Comment:
why 546? Is it worth defining a constant for this script and then if it changes again it will only need to be edited in one place.